apps/web/src/components/layout/EntryDetailLayout.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, PropsWithChildren } from "react"
+import { cn } from "@dotkomonline/ui"
+
+export interface EntryDetailLayoutProps {
+  title: string
+  type?: string
+  color?: "BLUE" | "GREEN" | "AMBER"
+}
+
+export const EntryDetailLayout: FC<PropsWithChildren<EntryDetailLayoutProps>> = ({ children, title, type, color }) => {
+  const borderColorClass = color ? `border-${color.toLowerCase()}-8` : "border-blue-8"
+  const textColorClass = color ? `text-${color.toLowerCase()}-11` : "text-blue-11"
```

Comment:
I don't think tailwind is clever enough to strip this at compile time, should do something like
```suggestion
export const EntryDetailLayout: FC<PropsWithChildren<EntryDetailLayoutProps>> = ({ children, title, type, color = 'BLUE' }) => {
  const borderColorClass = cn({
    "border-blue-8": color === 'BLUE',
    "border-green-8": color === 'GREEN',
    "border-amber-8": color === 'AMBER',
  })
  // same for textColorClass
```
})

Reply:
In my experience this should work, but I agree. Its harder to maintain without knowing exactly how tailwind works, and its way more clear to use your suggested and more intuitive method. I'll get it sorted.